doc/content/getting-started/migrating/migrating-from-v2.md

```diff
@@ -40,60 +117,28 @@ When using a **private V2 network server**, provided by The Things Industries, g
     discovery-address: <domain_id>.thethings.industries:1900
     ```
-    
-    **For windows users** make sure to use the `--config` flag with the full path to the configuration file: 
+
+    **For windows users** make sure to use the `--config` flag with the full path to the configuration file:
     ```
     ttnctl.exe --config fullpath/config.yml
     ```
 4. Get the access code needed for logging in: `https://<domain_id>.thethings.industries/users/authorize?client_id=ttnctl&redirect_uri=/oauth/callback/ttnctl&response_type=code`
-5. Login: 
+5. Login:
     ```bash
     $ ttnctl user login <access_code>
     ```
 6. You are now logged in your private network, you should now see a folder named `/.ttnctl` in your home directory with private files.
-    
-    > Optional: define an alias on ttnctl --config /home/<user_name>/.ttnctl/community.yml to go faster
-
-
-## Export End Devices from V2
 
-In this step, the end devices from {{% ttnv2 %}} will be exported in a JSON format that can then be parsed and imported by {{% tts %}}.
+    > Optional: define an alias on ttnctl --config /home/<user_name>/.ttnctl/community.yml to go faster
 
-The exported end devices contain the device name, description, location data, activation mode (ABP/OTAA), root keys and the AppEUI. They also contain the session keys, so your OTAA devices can simply keep working with {{% tts %}}.
 
 To get started, select the **AppID** and **AppEUI** of the application you want to export your end devices from:
 
 ```bash
 $ ttnctl applications select
 ```
 
-After selecting the application, make sure that you can list the available end devices:
-
-```bash
-$ ttnctl devices list
-```
-
-### Exporting Devices
-
-In order to export a single device, use the following command. The device will be saved to `device.json`.
-
-```bash
-$ ttnctl devices export "device-id" --frequency-plan-id EU_863_870 > device.json
-```
-
-Alternatively, you can export all the end devices with a single command and save them in `all-devices.json`.
-
-```bash
-$ ttnctl devices export-all --frequency-plan-id EU_863_870 > all-devices.json
-```
-
->**Note:** Change `EU_863_870` frequency plan from the command above to the frequency plan corresponding to your region. See [Frequency Plans]({{< ref "/reference/frequency-plans" >}}) for a list of supported Frequency Plans and their respective IDs.
-
->**Note:** Keep in mind that an end device can only be registered in one Network Server at a time. After importing an end device to {{% tts %}}, you should remove it from {{% ttnv2 %}}. For OTAA devices, it is enough to simply change the AppKey, so the device can no longer join but the existing session is preserved. Next time the device joins, the activation will be handled by {{% tts %}}.
-
-### Disable Exported End Devices on V2
-
-After exporting, make sure to clear the AppKey of your OTAA devices. This can be achieved with the following command:
+To clear the AppKey of an OTAA device, use the following command:
```

Comment:
This means that the session is still in V2, and that V2 is still sending downlink messages and managing the end device's MAC state, which may lead to problems